test/rest.test.js

```diff
@@ -722,6 +722,25 @@ describe('strong-remoting-rest', function() {
         .expect(204, done);
     });
 
+    it('should preserve non-200 status when responding with no content', function(done) {
+       var method = givenSharedStaticMethod(
+         function(ctx, cb) {
+           ctx.res.status(302);
+           cb();
+         },
+        {
+          accepts: [
+            { arg: 'ctx', type: 'object', http: {source: 'context' } }
```

Comment:
FYI: you can also use `{arg: 'res', type: 'object', http: { source: 'res' } }` and then access the response directly.